pkg/controllers/binding/common.go

```diff
@@ -306,7 +306,7 @@ func divideReplicasByJobCompletions(workload *unstructured.Unstructured, cluster
 	}
 
 	if found {
-		targetClusters = helper.SpreadReplicasByTargetClusters(int32(completions), clusters, nil)
+		targetClusters = helper.SpreadReplicasByTargetClusters(int32(completions), clusters, nil) // #nosec G115
```

Comment:
```suggestion
		targetClusters = helper.SpreadReplicasByTargetClusters(int32(completions), clusters, nil) // #nosec G115: integer overflow conversion int -> int32
```
It would be great to have an explanation.

Reply:
Sounds good. Updated.